glide.yaml

```diff
@@ -0,0 +1,12 @@
+package: main
+import:
+- package: gopkg.in
```

Comment:
I think the way this should work is that you should have a separate line for each _project_, like this:

```yaml
package: gopkg.in/tomb.v1
package: gopkg.in/yaml.v2
```

The easy way to test this would be to just do a `glide get gopkg.in/yaml.v2` and let it build the appropriate record.

The reason for this is that Glide sees _projects_ and _packages_. A project corresponds to the actual repo, and a package is the Go package or packages. Glide calls packages `subpackages` to sort of conceptually delineate that they are inside of a repository.